core/src/main/kotlin/com/imashnake/animite/core/extensions/ModifierExt.kt

```diff
@@ -1,13 +1,21 @@
 package com.imashnake.animite.core.extensions
 
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.displayCutoutPadding
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.platform.LocalConfiguration
 
 fun Modifier.given(
-    boolean: Boolean,
-    execute: Modifier.() -> Modifier
-): Modifier =
-    if (boolean) {
-        then(execute(Modifier))
-    } else {
-        this
-    }
+    boolean: Boolean, execute: Modifier.() -> Modifier
+): Modifier = if (boolean) {
+    then(execute(Modifier))
+} else {
+    this
+}
+
+fun Modifier.landscapeCutoutPadding() = composed {
```

Comment:
Modifier.given is silly here, we can directly do `if (LocalConfiguration.current.orientation == Configuration.ORIENTATION_LANDSCAPE) displayCutoutPadding() else this`